schema/i18n.json

```diff
@@ -0,0 +1,112 @@
+"interview_question": {
+  "title": "Internationalization (i18n)",
+  "prompts": [
+    {
+      "instructions": "Welcome to this programming exercise.\n\nYou will complete this exercise by editing the code in the sample.py file\nusing your favorite editor.\nEvery time you save the file, I will evaluate your code and let you know about\nnext steps.\n\nYour first task is to implement the abbreviate function. It takes a string as input\nand return an abbreviation of the string of the form <first character><length of the\nmiddle of the string><last character>. For example, \"internationalization\" should be\nabbreviated as \"i18n\".",
```

Comment:
I was just copying the format from the previous instance, but I can make these lists of strings if you'd like.